packages/external-db-mssql/lib/sql_exception_translator.js

```diff
@@ -7,14 +7,16 @@ const notThrowingTranslateErrorCodes = err => {
                 return new CollectionDoesNotExists('Collection does not exists')
             case 2705:
                 return new FieldAlreadyExists('Collection already has a field with the same name')
+            case 2627: 
+                return new ItemAlreadyExists(`Item already exists: ${err.message}`)
             case 4924:
                 return new FieldDoesNotExist('Collection does not contain a field with this name')
             case 2714:
                 return new CollectionAlreadyExists('Collection already exists')
             default:
                 console.log(err.number)
                 console.log(err)
```

Comment:
In my opinion, these prints are unnecessary, because the error middleware prints these errors.